src/qtgui/plotter.cpp

```diff
@@ -1298,7 +1298,7 @@ void CPlotter::drawOverlay()
         static const int fontHeight = fm.ascent() + 1;
         static const int slant = 5;
         static const int levelHeight = fontHeight + 5;
-        static const int nLevels = h / (levelHeight + slant);
+        int nLevels = h / (levelHeight + slant);
```

Comment:
Correct, this is a small, cosmetic fix that will make "excess" bookmark labels disappear not simply be written beyond the bottom of the plotter.

I was not able to reproduce the original problem for which I made this change. It can be disregarded, but I think it should remain part of the codebase so that when the plotter window does have a small height the labels don't overwrite other things (such as the band plan, although this awareness remains to be implemented).